scipy/sparse/tests/test_dok.py

```diff
@@ -0,0 +1,21 @@
+from numpy.testing import assert_array_equal
+from scipy.sparse import dok_array
+
+
+def test_fromkeys_default():
+    # test with default value
+    edges = [(0, 2), (1, 0), (2, 1)]
+    Xdok = dok_array.fromkeys(edges)
+    assert_array_equal(
+        Xdok.todense(),
```

Comment:
Use the `toarray()` method in favor of `todense()`.

Reply:
Done